src/ui/public/number_list/number_list.js

```diff
@@ -88,7 +88,17 @@ uiModules
             const list = self.getList();
             if (!list) return;
 
-            list.push(_.last(list) + 1);
+            function getNext(last, max) {
+              const next = last + 1;
+              if (next < max) {
+                return next;
+              }
+
+              return max - 1;
+            }
+
+            const next = getNext(_.last(list), self.range.max);
```

Comment:
With these changes, when adding the first item to an empty number list, the input default value that is displayed now is `max - 1` (`99` for Percentiles and `Infinity` for Percentile Ranks).

This is because previously, when adding the first item, the first thing pushed to the list would be `NaN`, because `_.last(list)` would be `undefined` in the empty array, i.e.:
```javascript
// list = [];
const last = _.last(list); // undefined
list.push(last + 1); // last + 1 = NaN;
// list = [NaN];
```
And `NaN` renders as nothing inside of the text input. This feels like a strange way for us to have implemented this in the first place, but perhaps we should update this to preserve that behavior so that we still have a blank input when adding the first item. Thoughts?


![screenshot 2019-01-10 14 07 34](https://user-images.githubusercontent.com/1608770/50997170-1d1abf00-14e1-11e9-9e4d-c079ffc71435.png)


![screenshot 2019-01-10 14 07 21](https://user-images.githubusercontent.com/1608770/50997171-1d1abf00-14e1-11e9-99fb-077cef7d471e.png)